TwilioVerifySDK/Info.plist

```diff
@@ -2,17 +2,25 @@
 <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
 <plist version="1.0">
 <dict>
-	<key>CFBundleDevelopmentRegion</key>
```

Comment:
Could you keep previous values and only add CFBundleVersion. SDK_* properties were removed

Reply:
Got It, I removed that extra info. Thanks 👍